go/examples/aws-lambda-source/receiver/secret_vars.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "// Code generated by $0; DO NOT EDIT." > secret_vars.go
+echo "package main" >> secret_vars.go
+echo "" >> secret_vars.go
+echo "var (" >> secret_vars.go
+echo -e "\tgoogleVisionAPIKey = \"$GOOGLE_VISION_API_KEY\"" >> secret_vars.go
```

Comment:
why code them at compile time?

Reply:
i cannot use env variables in lambda, so they have to be set at compile time.
to make sure they credentials are not commited into github, i use code generation

Reply:
does this doc help? https://docs.aws.amazon.com/lambda/latest/dg/env_variables.html

Reply:
thanks!  will fix